src/social-links.json

```diff
@@ -1,4 +1,35 @@
-{
+[
+  {
     "social_media_name": "link",
     "linkedin": "https://www.linkedin.com/in/"
-}
\ No newline at end of file
+  },
+  {
+    "social_media_name": "snapchat",
+    "snapchat": "https://www.snapchat.com/"
+  },
+  {
+    "social_media_name": "discord",
+    "discord": "https://www.discord.com/"
+  },
+  {
+    "social_media_name": "twitter",
+    "twitter": "https://www.twitter.com/"
+  },
+  {
+    "social_media_name": "reddit",
+    "reddit": "https://www.reddit.com/"
+  },
+   { 
+    "social_media_name": "spotify",
+    "spotify": "https://wwww.spotify.com/"
+  },
+  {
+    "social_media_name": "github",
+    "github": "https://www.github.com/"
+  },
+  {
+    "social_media_name":"gitlab",
+    "gitlab": "https://www.gitlab.com/"
```

Comment:
I guess, there's a confusion in the syntax 😅
All the links should be in one dictionary. Check syntax below 👇
```json
{
    "social_media_name": "link",
	"github": "https://github.com/"				// Example
    "linkedin": "https://www.linkedin.com/in/"	// Example
	
}
```

Like this.
The first line is just for the example, representing the **key** should contain the `social_media_name` and the **value** should contain `user profile prefix URL`.

Please add the links as specified in the above manner. Also, the keys should be in sorted alphabetically, so we can maintain an order.

I hope you are clear with the changes, if you have any confusions, feel free to ask.

_Happy Contributing_ 😊 🙌